package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@figmentio/elements",
```

Comment:
I accidentally published a new minor version with the console.log in below. 🤦 

At least this creates a good opportunity to test this functionality once this PR is merged in - I'll create a follow up PR that removes that console.log, and test auto-publishing.